server/plugins/Convolution.cpp

```diff
@@ -766,6 +775,10 @@ void StereoConvolution2L_Ctor(StereoConvolution2L *unit)
 		unit->m_prevtrig = 0.f;
 
 		SETCALC(StereoConvolution2L_next);
+
+		// initialize outputs
+		OUT0(0) = IN0(0);
+		OUT0(1) = IN0(1);
```

Comment:
`OUT0(1)` is definitely correct (see the recently written Server Plugin API page), but `IN0(1)` is not. Thanks for catching.